src/cclib/io/wfxwriter.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2017, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""A writer for wfx format files."""
+
+import os.path
+
+from . import filewriter
+
+class WFXWriter(filewriter.Writer):
+    """A writer for wfx files."""
+
+    required_attrs = ['atomcoords', 'atomnos', 'gbasis', 'charge', 'homos']
+    keyword = 'GTO'
```

Comment:
http://aim.tkgristmill.com/wfxformat.html 
AIMALL currently only supports Gaussian type orbitals. If ADF still does not support Gaussians, then wfx should not be written for it.

Maybe there should be a check to see if the orbitals are Gaussian type or not.